lib/SubscriptionSchedule.php

```diff
@@ -7,22 +7,19 @@
  *
  * @property string $id
  * @property string $object
- * @property mixed|null $billing_thresholds
```

Comment:
this is due to spec change

Reply:
yep normal that one, API version recently